src/common/containers/Creator.tsx

```diff
@@ -183,29 +204,70 @@ export class Creator extends React.Component<CreatorProps, CreatorState> {
         <Helmet title="Workshop" />
         <Title text="Workshop" />
 
-        <Button
-          color="secondary"
-          variant="contained"
-          style={{ marginLeft: 40, marginTop: 9 }}
-          onClick={this.handleClickNewCard}
-        >
-          <Icon style={{ marginRight: 10 }} className="material-icons">queue</Icon>
-          New Card
-        </Button>
-
-        {
-          cardOpenedForEditing &&
-          <CopyToClipboard text={this.permalinkUrl} onCopy={this.afterCopyPermalink}>
-            <Button
-              color="secondary"
-              variant="contained"
-              style={{ marginLeft: 10, marginTop: 9 }}
+        <div style={{ display: 'inline' }}>
+          <Button
```

Comment:
I wonder if these buttons can be pulled out into another component? Seems like its a ton of duplicated code